server/appengine/src/test/java/who/RefreshCaseStatsServletTest.java

```diff
@@ -34,17 +35,17 @@ public void testParse() throws UnsupportedEncodingException {
     // Test values in a cumbersome way.
     // TODO: Test against a snapshot of golden data.
```

Comment:
Here's something I was experimenting with today: Add a comment like this:
```
/*
The data in the file is this:
ADM0_NAME,CumCase,CumDeath,ISO_2_CODE,NewCase,NewDeath,date_epicrv
ACountry,100,10,AA,100,10,1608681600000
BCountry,101,11,BB,101,11,1608681600000
ACountry,150,15,AA,50,5,1608768000000
BCountry,152,17,BB,51,6,1608768000000
*/
```

Then `assertEquals(100+101+50+51, globalData.totalCases)`

This should make it more clear to understand by a tiny bit. Not needed everywhere, but in one or two tests may be useful.

(I can point you to the tool which generates that csv from the json if you don't have it at hand.)

Reply:
Added that to the `testPartialDay` test